pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -19,15 +20,17 @@ in pythonPackages.buildPythonApplication rec {
       --replace /etc $out/etc \
       --replace /lib/systemd $out/lib/systemd \
       --replace 'self.init_system = ""' 'self.init_system = "systemd"'
+
+    patchPhase
```

Comment:
It definitely does work, but don't ask me why :-) Anyway, I will change it to use `prePatch`.